xml/ay_create_control_file.xml

```diff
@@ -116,6 +117,13 @@
       similar, but not identical. This can be done by adjusting the profile
       using your favorite text/XML editor.
      </para>
+
+     <warning>
+      <para>
+       Beware that the profile might contain sensitive information.
+      </para>
+     </warning>
```

Comment:
Advisories need a title. Warnings should consist of two paragraphs. The first usually describes the danger, the second how to avoid it. For more info, refer to https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-admonition
```suggestion
     <warning>
     <title>Sensitive Data in Profiles</title>
      <para>
       Beware that the profile might contain sensitive information such as
       password hashes and registration keys.
      </para>
      <para>
       Carefully review the exported profiles and make sure to keep file permissions
       restrictive.
      </para>
     </warning>
```

Reply:
@cwickert Now it looks much better :) Thanks a lot!